test/system/252-quadlet.bats

```diff
@@ -198,6 +198,29 @@ EOF
     service_cleanup $QUADLET_SERVICE_NAME failed
 }
 
+@test "quadlet - oneshot" {
+    local quadlet_file=$PODMAN_TMPDIR/oneshot_$(random_string).container
+    cat > $quadlet_file <<EOF
+[Container]
+Image=$IMAGE
+Exec=echo INITIALIZED
```

Comment:
Can we do an `sh -c "echo INITIALIZED; sleep infinity"` and check that the container remains running once the service has started?

Reply:
Eh, the point of oneshot services is to exit. Otherwise they will never be considered "started".

Reply:
yes

Reply:
OK, I see wisdom in that decision.  In that case, it makes no sense to sleep.